### PR TITLE
Update zbus to v5

### PR DESCRIPTION
**Pull request by liff**
[zbus 5.0.0 release](https://github.com/dbus2/zbus/releases/tag/zbus-5.0.0)

Some fairly minor changes needed.

In zbus v5 the blocking API is feature-gated, so I wonder if the blocking secret service API should be similarly behind a feature?

### CODE REVIEW

**Review by complexspaces — APPROVED** [review]
If you wouldn't mind rebasing this on `master`, this looks good to go with the current changes 👍.

> In zbus v5 the blocking API is feature-gated, so I wonder if the blocking secret service API should be similarly behind a feature?

According to the `zbus` docs the `blocking-api` feature is enabled by default. I wouldn't mind adding the feature to parity match but I'm not entirely sure how it would interact with the existing features we have today. IIUC you would still need to select a `rt-` feature to use the blocking API, so it might just be additive and not create another dimension. Does that sound correct to you as well?

Long story short I'm in favor of the feature flag for consistency as long as it doesn't make the `rt-tokio-crypto-rust` problem worse.

**Comment by liff** [comment]
Rebased.

> IIUC you would still need to select a rt- feature to use the blocking API, so it might just be additive and not create another dimension

I don’t think there’d be another dimension. Disabling the `blocking-api` feature would basically just omit the `secret_service::blocking` module, if I remember correctly. It’s not something that needs dealing with, but could probably be done if there turn out to be useful aspects to it.

**Review by complexspaces — APPROVED** [review]
Got it, thanks for helping clarify. Since it doesn't add that much code to the build and doesn't bring in more dependencies having it enabled I'm OK leaving it be for now and seeing if the feature request comes up.
